src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,43 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        this.keys = (K[]) new Object [MAX_NUMBER];
+        this.values = (V[]) new Object [MAX_NUMBER];
+        size = 0;
+    }
```

Comment:
```suggestion
    private int size;

    public StorageImpl() {
        this.keys = (K[]) new Object [MAX_NUMBER];
        this.values = (V[]) new Object [MAX_NUMBER];
    }
```
class fields always have default values